docker/pulsar-all/pom.xml

```diff
@@ -119,57 +111,73 @@
             <executions>
               <execution>
-                <id>default</id>
+                <id>docker-build</id>
+                <phase>package</phase>
                 <goals>
-                  <goal>build</goal>
-                </goals>
-              </execution>
-              <execution>
-                <id>add-no-repo-and-version</id>
-                <goals>
-                  <goal>tag</goal>
-                </goals>
-                <configuration>
-                  <repository>pulsar-all</repository>
-                  <tag>${project.version}</tag>
-                </configuration>
-              </execution>
-              <execution>
-                <id>add-no-repo-and-latest</id>
-                <goals>
-                  <goal>tag</goal>
+                  <goal>exec</goal>
                 </goals>
                 <configuration>
-                  <repository>pulsar-all</repository>
-                  <tag>latest</tag>
+                  <environmentVariables>
+                    <DOCKER_BUILDKIT>1</DOCKER_BUILDKIT>
+                  </environmentVariables>
+                  <executable>docker</executable>
+                  <workingDirectory>${project.basedir}</workingDirectory>
+                  <arguments>
+                    <argument>build</argument>
+                    <argument>--no-cache</argument>
+                    <argument>--build-arg</argument>
+                    <argument>PULSAR_TARBALL=target/pulsar-server-distribution-${project.version}-bin.tar.gz</argument>
+                    <argument>-t</argument>
+                    <argument>pulsar-all:${project.version}</argument>
+                    <argument>-t</argument>
+                    <argument>pulsar-all:latest</argument>
+                    <argument>-t</argument>
+                    <argument>${docker.organization}/pulsar-all:${project.version}</argument>
+                    <argument>-t</argument>
+                    <argument>${docker.organization}/pulsar-all:latest</argument>
+                    <argument>.</argument>
+                  </arguments>
                 </configuration>
               </execution>
               <execution>
-                <id>tag-and-push-latest</id>
+                <id>docker-push</id>
+                <phase>deploy</phase>
                 <goals>
-                  <goal>tag</goal>
-                  <goal>push</goal>
+                  <goal>exec</goal>
                 </goals>
                 <configuration>
-                  <repository>${docker.organization}/pulsar-all</repository>
-                  <tag>latest</tag>
+                  <environmentVariables>
+                    <DOCKER_BUILDKIT>1</DOCKER_BUILDKIT>
+                  </environmentVariables>
+                  <executable>docker</executable>
+                  <workingDirectory>${project.basedir}</workingDirectory>
+                  <arguments>
+                    <argument>buildx</argument>
```

Comment:
I think `buildx` requires `BuildKit container` in order to build for the multi platforms. Don't we need to exec(or check) `docker buildx create --use --driver docker-container` before this `buildx` execution?